package.json

```diff
@@ -39,22 +39,24 @@
     "jest-puppeteer": "^5.0.4",
     "jest-runtime": "^27.2.1",
     "puppeteer": "^3.3.0",
+    "tar": "^6.1.11",
     "ts-jest": "^27.0.5",
     "typedoc": "^0.22.18",
     "typedoc-plugin-merge-modules": "^3.1.0",
     "typescript": "^4.7.4",
     "uuid": "^8.3.2",
-    "yargs": "^17.2.1",
-    "cmake-js": "^6.3.2",
-    "tar": "^6.1.11"
+    "yargs": "^17.2.1"
   },
   "dependencies": {
     "@aws-sdk/util-utf8-browser": "^3.109.0",
+    "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@httptoolkit/websocket-stream": "^6.0.0",
     "axios": "^0.24.0",
+    "cmake-js": "7.1.1",
```

Comment:
why do we pin to `7.1.1` instead of using `^7.1.1`?

Reply:
7.2.x bumps axios to a version that causes import errors on x86 node v12